ts/Data/DataStore.ts

```diff
@@ -80,6 +85,21 @@ abstract class DataStore {
 
     }
 
+    colsToDataTable(cols: Array<Array<Highcharts.DataValueType>>, headers: string[] = []): DataTable {
+        const table = new DataTable();
+        const noOfCols = cols.length;
+        if (noOfCols) {
+            const noOfRows = cols[0].length;
+            for (let i = 0; i < noOfRows; ++i) {
+                const row = new DataRow();
+                for (let j = 0; j < noOfCols; ++j) {
+                    row.insertColumn((headers.length ? headers[j] : uniqueKey()), cols[j][i]);
+                }
+                table.insertRow(row);
+            }
+        }
+        return table;
+    }
```

Comment:
In my opinion this should not be part of the DataStore class. It is not related to store functionality, but more to parsing. It should better be outsourced into a new `DataParser` module file similar to the DataConverter class.

Beside that the variable names should be clearer to avoid confusion. Instead of `noOfRows` use `rowsLength`. Use `columns` instead of `cols`. (See https://en.wikipedia.org/wiki/Col)